src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+/**
+ * @author user
+ * This class is an abstract class for Email, Phone, Address
+ */
```

Comment:
There is no need to state the subclasses it is used for. Instead, state what a user can do with the interface.